### ISSUE TITLE
Android - General - “Share” and “About” Act as Links when It should be as a Button

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#9]
TalkBack reads the text “Share” and “About” as a link when I would expect them to be read as buttons because they do a function. While the “Bookmarks” and “Search” text is read as a link which makes sense because they go to another screen.

**Comment by KyleLivingstonDOL** [comment]
It looks like this is done. UAT for Sprint 7 (which is ongoing right now) will start on June 10.

**Comment by jfc3-dol** [comment]
TalkBack is pronouncing the items as expected so going to close the issue,
